### ISSUE TITLE
Right-click context menu working, but not left click

### ISSUE THREAD

**Issue by MartinAyla** [sbussetti/deluge-siphon#32]
Hi

For the "Download" button on this site, only the right-click menu works, left-clicking just downloads the file normally (I have left-click option check in deluge-siphone settings of course).

Any ideas?

Site: http://720pier.ru/viewtopic.php?f=34&t=7888

**Comment by sbussetti** [comment]
I'm working on this right now -- just wanted to yank the link out of comments.. you know for science.  I have it.

**Comment by sbussetti** [comment]
Okay Version 0.70.3 updates the torrent matching regex to work on 720pier, however I ran into a new problem with that site.

The way this extension works is, delugesiphon tells your deluge server about the torrent, and then your deluge server downloads the .torrent file from the site you're using (in this case 720pier), however 720pier is blocking the deluge server user agent from talking to their site (which seems silly).  I'm going to continue to work on  this one.

**Comment by MartinAyla** [comment]
Thanks a lot for working on this, and thanks for a great extension!

**Comment by MartinAyla** [comment]
Any luck on this? :)

**Comment by sbussetti** [comment]
@MartinAyla yeah actually.  I spoke with one of the admins on the site last week and got some insight.  The issue is actually between 720pier and deluge, not between delugesiphon and 720pier, however, I'm working on a plugin for deluge that should resolve it.  I have some code written but 720pier has been inaccessible for the past few days so I haven't been able to continue testing.  Fingers crossed it comes back!
